Add 4 new questions to GitHub Foundations around GitHub administration based on Microsoft Learn guides and GitHub official documentation. (#209)
FidelusAleksander/ghcertified: content/questions/foundations/question-107.md

---
archetype: "questions"
title: "Question 107"

> question: "Which permissions can setup billing or assign billing managers for their organization(s)? (Select 2)"

Reviewer: ```suggestion
question: "Who can setup billing or assign billing managers for an organization? (Select two.)"
```

draft: false
---

> > https://learn.microsoft.com/en-us/training/modules/github-introduction-administration/2-what-is-github-administration

Reviewer: Maybe [this](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/adding-a-billing-manager-to-your-organization) is a better source? Also seems like existing Billing Managers can also invite additional people to become billing managers

Author: Gotcha, I was going off the study guides that GitHub resources linked here: https://resources.github.com/learn/certifications.  But it is good to cross reference then official GitHub instead.

- [ ] Owner of the repository

> > This is not mentioned as a permission level for GitHub administration.

Reviewer: This is not helpful. Re-phrase or delete
```suggestion
```

> > This is not mentioned as a permissiosn level for GitHub administration.

Reviewer 2: ```suggestion
> This is not mentioned as a permission level for GitHub administration.
```

- [ ] Team maintainer at the team level

> > Billing is not mentioned at the team level.

Reviewer: This is not helpful. Re-phrase or delete

```suggestion
> Being a team maintainer does not grant billing access
```

- [ ] Admin in at the team level

> > Billing is not mentioned at the team level.

Reviewer: This is not helpful. Re-phrase or delete
```suggestion
```

---
archetype: "questions"
title: "Question 107"

> question: "Who can setup billing or assign billing managers for an organization? (Select two.)"

Reviewer: ```suggestion
question: "Who can setup billing or assign billing managers for an organization? (Select three.)"
```
